scripts/openshift-helpers.sh

```diff
@@ -85,7 +87,9 @@ function openshift_terminate() {
 	rm ${HPO_DEPLOY_MANIFEST}
 	echo
 
-	echo
-	echo "Removing HPO namespace"
-	kubectl delete ns ${hpo_ns}
+	if [ ${hpo_ns} != "monitoring" ]; then
```

Comment:
This should be `openshift_monitoring`. However I'd suggest that we only delete if it is `openshift_tuning`. This way if a user has created a separate ns where they would like to install HPO, they will take care of cleaning it up themselves. By default, we will create `openshift_tuning` and delete it ourselves.